changelog/@unreleased/pr-5140.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: fix
+fix:
+  description: The No op lock watch event cache now correctly tracks the latest known
```

Comment:
nit: `no-op` or `no op` (or even `No Op`?) - something seems strange with the capitalisation